DEV-3/DEV-3/NumbersConversion.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DEV_3
+{
+    /// <summary>
+    /// Conversion a number to the selected number system
+    /// </summary>
+    class NumbersConversion
+    {
+        int number;
+        int scaleOfNotation;
+        public NumbersConversion (int arg0, int arg1)
+        {
+            number = arg0;
+            scaleOfNotation = arg1;
+        }
+
+        /// <summary>
+        /// Conversion a number to the selected number system
+        /// </summary>
+        /// <returns> Converted number</returns>

```

Comment:
It is reasonable to return the converted number as string, but not massive of ints